### ISSUE TITLE
Terms and Conditions

### ISSUE THREAD

**Issue by nickburne** [NickBurneConsulting-GivePanel/givepanel#37]

**Comment by nickburne** [comment]
We have terms and conditions from solicitor. Nick made comments and is waiting to hear. Should have a final version by the end of this week.

**Comment by nickburne** [comment]
I have the final version of this now.

On the sign up form there is an opt-in checkbox for the terms and conditions. The terms and conditions need to be a link that opens a modal window with the terms and conditions inside.

**Comment by shramee** [comment]
Is this okay for terms and conditions page..?

![image](https://user-images.githubusercontent.com/11048263/52303102-19ca0480-29b5-11e9-9d29-71c7a12ce7e4.png)

When there's more text than available space... it's scrollable...

![image](https://user-images.githubusercontent.com/11048263/52303228-6a416200-29b5-11e9-905a-3cc3ed1d1cb9.png)

**Comment by nickburne** [comment]
@jonrandahl - we need to insert the actual wording next :)

**Comment by jonrandahl** [comment]
The actual wording is on the page as per the latest word document provided. Header styles are present, as are other minor typographical styles as well; however, we will need further styling to override the Bootstrap reset for nested lists unless we want to put everything into a [reactstrap list group](https://reactstrap.github.io/components/listgroup/).

**Comment by jonrandahl** [comment]
Over to you @nickburne for approval.

**Comment by nickburne** [comment]
@jonrandahl - problems with the terms and privacy.

Recorded a Loom video for you...

https://www.useloom.com/share/ed786eabacd64ffaad0f5d9c96c9f9d8

**Comment by jonrandahl** [comment]
1. Terms & Conditions with numbers
2. External links to open in new tab/windows

**Comment by jonrandahl** [comment]
Final version:

![image](https://user-images.githubusercontent.com/39102/52806690-8c0baa80-3081-11e9-8a06-cbd35acdffca.png)

**Comment by nickburne** [comment]
Numbers of terms and conditions don't correspond with word document. e.g. there is no 1.34 on the word doc...

Please check numbers match exactly. This is a legal document and so needs to be thoroughly checked! Detail... :)

**Comment by jonrandahl** [comment]
Thanks chief, will do! I’ll make this one first thing!

**Comment by jonrandahl** [comment]
Updated styling to apply correct counters and double checked by printing out the document and verifying for each entry.

![image](https://user-images.githubusercontent.com/39102/53090549-b3da9280-3507-11e9-8f99-ba15f1758a0d.png)

@nickburne there is an Error in the supplied word document, for item 9.5 which states:

`Error! Reference source not found.`

Not sure what this is from?

I've pushed this to both Staging and Demo for testing and will push to live on next update.
